jsk_recognition_msgs/CMakeLists.txt

```diff
@@ -2,9 +2,6 @@ cmake_minimum_required(VERSION 2.8.3)
 project(jsk_recognition_msgs)
 find_package(catkin REQUIRED
   std_msgs sensor_msgs geometry_msgs message_generation pcl_msgs jsk_footstep_msgs)
-if(NOT $ENV{ROS_DISTRO} STREQUAL "indigo") # on noetic it needs catkin_install_python to support Python3 and it does not work on indigo for some reason...
-  catkin_python_setup()
```

Comment:
I may not understand fully, but I think this `catkin_python_setup` is needed to call `catkin_install_python` here:
https://github.com/jsk-ros-pkg/jsk_recognition/blob/a3283f82a9d5e58019ea49644c1d50ebf5d3407b/jsk_recognition_msgs/CMakeLists.txt#L159
and this `catkin_install_python` is needed to manage python shebang depending on python version (2 or 3) used by ROS system.
cf. https://github.com/jsk-ros-pkg/jsk_recognition/pull/2743

Reply:
I think `catkin_setup_python` is not required because there is no python libraries to be installed

Reply:
@mqcmd196 Sorry, you are correct...

Reply:
> I think this catkin_python_setup is needed to call catkin_install_python here

Yes, that was the mistake made in the original commit.

The documentation explains that **scripts** and **modules** use independent mechanisms in catkin, so the call to `catkin_python_setup` is not needed for scripts:
http://docs.ros.org/en/melodic/api/catkin/html/howto/format2/installing_python.html